CMIP6_source_id.json

```diff
@@ -740,44 +750,47 @@
             "label_extended":"CNRM-CM6-1",
             "model_component":{
                 "aerosol":{
-                    "description":"climatological fields computed by TACTIC (v2.0)",
-                    "nominal_resolution":""
+                    "description":"prescribed monthly fields computed by TACTIC_v2 scheme",
+                    "nominal_resolution":"250 km"
                 },
                 "atmos":{
-                    "description":"Arpege 6.2 (Tl127; 91 levels; top level 78.4 km)",
-                    "nominal_resolution":""
+                    "description":"Arpege 6.3 (Tl127, 24572 x 1; XXX x XXX longitude/latitude; 91 levels; top level 78.4 km)",
```

Comment:
The entry `24572 x 1; XXX x XXX longitude/latitude;` needs to be updated to a single correct value